src/java/main/org/apache/zookeeper/server/persistence/FileTxnLog.java

```diff
@@ -199,14 +199,16 @@ public synchronized boolean append(TxnHeader hdr, Record txn)
                 LOG.warn("Current zxid " + hdr.getZxid()
```

Comment:
I like the suggestion here. Though ZK does not enforce such style, early exists is my preference for reasons documented here: https://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code Though this change can be done separately as it's not an existing stylish issue.

>> BTW,why do I see so many flaky changes?

Not sure about what flaky changes you are referring to? @maoling